br/pkg/lightning/mydump/router.go

```diff
@@ -324,6 +324,9 @@ func (p regexRouterParser) Parse(r *config.FileRouteRule, logger log.Logger) (*R
 			if err != nil {
 				return err
 			}
+			if result.Type == SourceTypeParquet && compression != CompressionNone {
+				return errors.Errorf("can't support whole compressed parquet file, should compress parquet files by choosing correct parquet compress writer, path: %s", r.Path)
```

Comment:
What does this workaround mean here? I remember we support compressed parquet files.

Reply:
We support compressed binary data in parquet files, but we don't support compressed parquet files like `parquet.gz`.